msgconv/to-whatsapp.go

```diff
@@ -101,8 +101,7 @@ func (mc *MessageConverter) ToWhatsApp(
 	var meta waMsgApplication.MessageApplication_Metadata
 	if replyTo := mc.GetMetaReply(ctx, content); replyTo != nil {
 		meta.QuotedMessage = &waMsgApplication.MessageApplication_Metadata_QuotedMessage{
-			StanzaID:  replyTo.ReplyMessageId,
-			RemoteJID: mc.GetData(ctx).JID().String(),
+			StanzaID: replyTo.ReplyMessageId,
 			// TODO: this is hacky since it hardcodes the server
 			// TODO 2: should this be included for DMs?
```

Comment:
@tulir does this clear this TODO?

Reply:
Probably not, the TODO is for the `Participant` field

Reply:
Oh actually it might clear the second one if you've confirmed DMs are supposed to have that field. I think on WhatsApp the same field is meant for groups (and RemoteJID is used for DM recipient or group ID)

Reply:
From the screenshot in the PR description it seems that the `Participant` field is indeed required.